### PR TITLE
Add intel manual section talking about APERF's undefinedness.

### PR DESCRIPTION
**Pull request by vext01**
OK?

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -1632,3 +1632,10 @@ @article{antoch97effect
     volume = {60},
     journal = {Journal of Statistical Planning and Inference}
 }
+
+@manual{intel2017pstate,
+    title        = {Intel® 64 and IA-32 Architectures Software Developer’s Manual: P-State hardware coordination},
```
Comment (ltratt) [inline softdev.bib]: We don't need to use the ® symbol, so let's remove it.

Comment (ltratt) [inline softdev.bib]: Do you need to force capitalisation of some of those things with `{...}`?

**Comment by vext01** [comment]
I title cased the title (it doesn't appear to need explicit {}), and also used `author` instead of `organization`, as the bibtex style was doing the wrong thing. We now have:

ref

Let me squash if it's OK :)

File: softdev.bib (hunk 1 above)
Comment (vext01) [inline softdev.bib]: Killed (r), see below about {}

**Comment by ltratt** [comment]
Looks good! Squash away.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
don't forget this one :eagle:

**Pull request merged**